openpype/hosts/houdini/plugins/create/create_alembic_camera.py

```diff
@@ -47,3 +49,9 @@ def create(self, subset_name, instance_data, pre_create_data):
         self.lock_parameters(instance_node, to_lock)
 
         instance_node.parm("trange").set(1)
+
+    def get_network_categories(self):
```

Comment:
should we add `def get_network_categories(self)` on plugin.HoudiniCreator and only overload the creators that need some extra categories?
```
def get_network_categories(self):
   super().get_network_categories().extend(hou.objNodeTypeCategory())
```
Alternatively, I'm also thinking whether it'd be worth making these a class variable instead so you don't need to initialize the creator to get the info:
```
class CreateAlembicCamera(plugin.HoudiniCreator):
    """Single baked camera from Alembic ROP."""
    network_categories = plugin.HoudiniCreator.network_categories + [hou.objNodeTypeCategory()]

```

Reply:
I've kept them functions for now but I did move the implementation.